src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
+
```

Comment:
Inside of `src` create a `types` folder for all TS interfaces. Each interface should have a file for it. So create as many files  as you need.
For example `types/User.d.ts` should contain only `interface User`